test/node-unit/cli/config.spec.js

```diff
@@ -109,11 +109,15 @@ describe('cli/config', function() {
 
     describe('when config file parsing fails', function() {
       beforeEach(function() {
-        sinon.stub(parsers, 'yaml').throws();
+        sinon.stub(parsers, 'yaml').throws('goo.yaml is unparsable');
       });
 
       it('should throw', function() {
-        expect(() => loadConfig('goo.yaml'), 'to throw', /failed to parse/);
+        expect(
+          () => loadConfig('goo.yaml'),
+          'to throw',
+          'goo.yaml is unparsable'
```

Comment:
I'm kind of unsure what exactly we should be testing here.

the original code's assertion says that whenever `parsers.yaml()` throws, the error thrown by `loadConfig()` should contain the text `/failed to parse/`.

with this change, this assertion says that the `message` prop of the `Error` thrown by `parsers.yaml()` should wind up in an `Error` thrown by `loadConfig()`.  this reflects that the message is somehow re-used--which the original code's assertion _should_ have done.  so this is an improvement.

but...

- should we assert it is _not_ the _same_ `Error` as thrown by `parsers.yaml()`? 
- should we assert `loadConfig()` passes the original error message thrown by `parsers.yaml()` into  `createUnparsableFileError()`?
- should we assert the thrown error has the appropriate `code` prop?
- should we stub `createUnparsableFileError()` to return an `Error` with our _own_ `code` prop?

I don't really know the answers here yet, but I'm happy to discuss them if you have the time.  I don't think it's worth blocking the PR for, either way

Reply:
When I first encountered this test as a failure when I changed this code, it was a bit puzzling to just see `throws()` and `failed to parse` expected in the failure since just the empty `throws()` itself did not seem to indicate any information about what to expect. I think now with the additional changes from your suggestions to add the "unable to parse....", it seems to be clearer where this error came from.